### PR TITLE
Add more JUnit tests

### LINKED ISSUE #159: Add more JUnit tests
For SampleDataUtil.java, AddressBook.java, Tag.java, ModelManager.java.

### PR DESCRIPTION
**Pull request by gohsl99**
Fixes #159

### CODE REVIEW

File: src/test/java/seedu/address/model/AddressBookTest.java [hunk 1]
```diff
@@ -75,17 +90,69 @@ public void hasPerson_personWithSameIdentityFieldsInAddressBook_returnsTrue() {
         assertTrue(addressBook.hasPerson(editedAlice));
     }
 
+    @Test
+    public void removePerson_personToRemove_personRemoved() {
```
Comment (winstonlimjy) [inline src/test/java/seedu/address/model/AddressBookTest.java:94]: perhaps renaming to removePerson_personToRemove_success() is better?

File: src/test/java/seedu/address/model/AddressBookTest.java [hunk 2]
```diff
@@ -75,17 +90,69 @@ public void hasPerson_personWithSameIdentityFieldsInAddressBook_returnsTrue() {
...
+        addressBook.addPerson(FIONA);
+        assertTrue(addressBook.hasPerson(FIONA));
+        addressBook.removePerson(FIONA);
+        assertFalse(addressBook.hasPerson(FIONA));
+    }
+
+    @Test
+    public void setPerson_updatedPerson_personUpdated() {
```
Comment (winstonlimjy) [inline src/test/java/seedu/address/model/AddressBookTest.java:102]: same goes for here! just a suggestion maybe could change to setPerson_updatePerson_success()

**Comment by winstonlimjy** [comment]
Good coverage of tests. Maybe could consider amending some method names.

File: src/test/java/seedu/address/model/AddressBookTest.java (hunk 1 above)
Comment (gohsl99) [reply src/test/java/seedu/address/model/AddressBookTest.java]: ok, changed the name.

**Pull request merged**